tests/IceRpc.Tests.Common/ColocInvoker.cs

```diff
@@ -66,7 +66,7 @@ public async Task<IncomingResponse> InvokeAsync(OutgoingRequest request, Cancell
             }
             else
             {
-                var pipe = new Pipe();
+                var pipe = new Pipe(new PipeOptions(pauseWriterThreshold: 0));
```

Comment:
This prevents the test from hanging. If we send a large payload with the ColocInvoker, the default pause writer threshold is 16k.